src/OrbitSshQt/TaskTest.cpp

```diff
@@ -64,6 +64,10 @@ TEST_F(SshTaskTest, Stdout) {
   QSignalSpy ready_read_signal{&task, &orbit_ssh_qt::Task::readyReadStdOut};
 
   EXPECT_THAT(WaitForWithTimeout(task.Start()), YieldsResult(HasNoError()));
+  // If we haven't yet received the `finished` signal, let's wait for it.
+  if (finished_signal.empty()) {
+    EXPECT_TRUE(finished_signal.wait());
+  }
```

Comment:
Note that this check was always missing. Changing the implementation of `WaitForWithTimeout` just slightly changed the timing and made this test fail. It just happened to work before hand.

Same in the `Stderr` test below.